pds4indextools/template_pds.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1L00.sch"
+    schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
+$IF(Product_Ancillary)   
+<Product_Ancillary xmlns="http://pds.nasa.gov/pds4/pds/v1"
+ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://pds.nasa.gov/pds4/pds/v1 https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1L00.xsd">
+$END_IF
+$IF(Product_Metadata_Supplemental)   
+<Product_Metadata_Supplemental xmlns="http://pds.nasa.gov/pds4/pds/v1"
+ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://pds.nasa.gov/pds4/pds/v1 https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1L00.xsd">
+$END_IF
+    <Identification_Area>
+        <logical_identifier>$logical_identifier$</logical_identifier>                          
+        <version_id>$version_id$</version_id> <!--user input, optional-->
+        <title>$title$</title>           <!--user input, optional-->
+        <information_model_version>1.21.0.0</information_model_version>
+        <product_class>Product_Ancillary</product_class>
+   $IF(Citation_Information)
+       <Citation_Information>          <!--user input, optional-->
+            <author_list>$Citation_Information['author_list']$</author_list> <!--user input, optional-->
+            <editor_list>$Citation_Information['editor_list']$</editor_list> <!--user input, optional-->
+            <publication_year>$Citation_Information['publication_year']$</publication_year> <!--user input, optional-->
+            <doi>$Citation_Information['doi']$</doi> <!--user input, optional-->
+        $FOR(Citation_Information['keyword'])
+            <keyword>$VALUE$</keyword> <!--user input, optional-->
+        $END_FOR
+            <description>$Citation_Information['description']$</description> <!--user input, optional-->
+        $IF(Citation_Information.get('Funding_Acknowledgement']))
+            <Funding_Acknowledgement><!--user input, optional-->
+                <funding_source>$Funding_Acknowledgement['funding_source']$</funding_source> <!--user input, optional-->
+                <funding_year>$Funding_Acknowledgement['funding_year']$</funding_year> <!--user input, optional-->
+                <funding_award>$Funding_Acknowledgement['funding_year']$</funding_award> <!--user input, optional-->
```

Comment:
Are these always the same year?

Reply:
Ah, copy-paste typo.

Reply:
I don't understand the question, but it seems like the second instance of `funding_year` should be `funding_award`, so that it matches the attribute name. 

These are three attributes describing a single source of funding that is being acknowledged.  Three values for a single award might be NASA, 2024, and CDAP.